app/src/components/ConvictionVisuals.js

```diff
@@ -292,15 +251,19 @@ export function ConvictionCountdown({ proposal, shorter }) {
 }
 
 export function ConvictionTrend({ proposal }) {
+  const { appState } = useAragonApi()
+  const {
+    globalParams: { alpha },
+  } = appState
```

Comment:
Ah in this cases where we only use the app state we could use `useAppState` and avoid extra lines

```suggestion
  const {  globalParams: { alpha }  } = useAppState()

```